env.example

```diff
@@ -1,2 +1,4 @@
 MONGO_URI="mongodb+srv://mappypals:ZozaDZQ7FDKmZZIq@mappypals-hzxcd.mongodb.net/test?retryWrites=true"
-JWT_SECRET="w23Q$8$7sK$aQod83YQv2#29vOV2#0V"
\ No newline at end of file
+JWT_SECRET="w23Q$8$7sK$aQod83YQv2#29vOV2#0V"
+EMAIL_ADDRESS="team.mappypals@gmail.com"
+EMAIL_PASSWORD="mappypalsadmin"
```

Comment:
The file is not formatted try running `npm run dev:format`

Reply:
I have formatted it now.